src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj

```
@@ -54,6 +54,7 @@
     <PackageReference Include="System.Runtime" Version="4.3.1" />
     <PackageReference Include="System.Runtime.InteropServices" Version="4.3.0" />
     <PackageReference Include="XliffTasks" Version="1.0.0-beta.19252.1" PrivateAssets="all" />
+	<PackageReference Include="K4os.Compression.LZ4" Version="1.2.1-beta" />
```

Should we use the stable version instead? https://www.nuget.org/packages/K4os.Compression.LZ4/1.1.11

You might also swap this tab for spaces:
```suggestion
    <PackageReference Include="K4os.Compression.LZ4" Version="1.2.1-beta" />
```

---

I'm using the stable version in my fastdevv2 branch, so we should probably stick to the same version https://github.com/xamarin/monodroid/pull/1090/files#diff-3636cde4a1d6c35c47487adff92dae2aR43 :)